provider/blob/benches/auxkey_bench.rs

```diff
@@ -94,7 +94,7 @@ fn make_blob_v1() -> Vec<u8> {
         .with_locales_and_fallback([LocaleFamily::FULL], Default::default())
         .export(&Baked, exporter)
         .unwrap();
-    assert_eq!(blob.len(), 684197);
+    assert_eq!(blob.len(), 450725);
```

Comment:
FYI, the other test right below this one goes to 241278

Reply:
thanks, everything should pass now